web-api/src/v1/getCaseLambda.test.js

```diff
@@ -73,7 +72,7 @@ describe('getCaseLambda', () => {
     );
   });
 
-  it('returns 403 when the user is not authorized and the case is found', async () => {
+  it('returns 200 when the user is not associated and the case is found', async () => {
```

Comment:
Commented this in the wrong place previously (Flexion's PR):

it'd be nice to clarify that it returns only the publicly visible version of the case - this seems like something we should ensure IRS is aware of so they can handle it, too. cc @adunkman @mmarcotte